src/energy_grid.F90

```diff
@@ -14,63 +14,17 @@ module energy_grid
 contains
```

Comment:
I suppose that could pay off when a particle moves between cells where its energy doesn't change but the temperature does (and cross sections need to be recalculated for the same nuclides). Would be an interesting experiment once we're at that point.